shared/message/platform/webots/messages.proto

```diff
@@ -97,22 +97,13 @@ message Message {
 }
 
 /// NUbots specific data sent from our own controllers rather than the official RoboCup Webots controller
-message OdometryGroundTruth {
+message RobotPoseGroundTruth {
     /// Indicates if this message exists
     bool exists = 1;
-    /// Affine3d transform from odometry (world) space to the robot (torso) space.
-    /// It measures the world in robot space
-    mat4 Htw = 2;
-    /// Velocity of the robot torso in the world space
-    vec3 vTw = 3;
-}
-
-/// NUbots specific data sent from our own controllers rather than the official RoboCup Webots controller
-message LocalisationGroundTruth {
-    /// Indicates if this message exists
-    bool exists = 1;
-    /// Isometry3d transform from localisation (world) space to field space
-    mat4 Hfw = 2;
+    /// Affine3d transform from odometry (field) space to the robot (torso) space.
+    mat4 Hft = 2;
+    /// Velocity of the robot torso in the field space
+    vec3 vTf = 3;
```

Comment:
I feel like this shouldn't live in this file, but instead in raw sensors or its own support::GroundTruth file.

Reply:
This needs to be synced with the webots codebase so I think it's better to keep it here. The values are then populated in another message locally `localisation::RobotPoseGroundTruth`